test/shared/src/test/scala/monocle/LensSpec.scala

```diff
@@ -27,6 +27,9 @@ class LensSpec extends MonocleSuite {
   val s = Lens[Example, String](_.s)(s => ex => ex.copy(s = s))
   val p = Lens[Example, Point](_.p)(p => ex => ex.copy(p = p))
 
+  implicit def ev = IndexedStoreT.storeTComonadStore[Id.Id, String]
```

Comment:
It seems that *scalaz* contains only two instances of `ComonadStore`: `StoreT` and `TracedT`. Honestly, I've never dealt with the second one, but it doesn't appear to be very practical for the case of *Lenses* (for instance, having [`copoint`](https://github.com/scalaz/scalaz/blob/series/7.3.x/core/src/main/scala/scalaz/TracedT.scala#L20) using `Monoid.zero` to feed the inner function seems odd). Given this situation, do you think it would be preferable to work directly with `Store`?